fluss-common/src/test/java/com/alibaba/fluss/utils/concurrent/FutureUtilsTest.java

```diff
@@ -389,4 +397,143 @@ void testOrTimeout() {
                 .withCauseInstanceOf(TimeoutException.class)
                 .withMessageContaining(expectedErrorMessage);
     }
+
+    @Test
+    void testCompletedVoidFuture() {
+        final CompletableFuture<Void> future = FutureUtils.completedVoidFuture();
+        assertThatFuture(future).eventuallySucceeds().isNull();
+    }
+
+    @Test
+    void testCompleteFromCallable() {
+        final CompletableFuture<String> successFuture = new CompletableFuture<>();
+        FutureUtils.completeFromCallable(successFuture, () -> "Fluss");
+        assertThatFuture(successFuture).eventuallySucceeds().isEqualTo("Fluss");
+
+        final CompletableFuture<String> failureFuture = new CompletableFuture<>();
+        FutureUtils.completeFromCallable(
+                failureFuture,
+                () -> {
+                    throw new RuntimeException("mock runtime exception");
+                });
+        assertThatFuture(failureFuture)
+                .eventuallyFailsWith(ExecutionException.class)
+                .withCauseInstanceOf(RuntimeException.class)
+                .withMessageContaining("mock runtime exception");
+    }
+
+    @Test
+    void testRunIfNotDoneAndGet() throws Exception {
+        FutureUtils.runIfNotDoneAndGet(null);
+        Runnable task = () -> System.out.println("Fluss");
+        FutureTask<String> futureTask = new FutureTask<>(task, "Hello Fluss");
+        FutureUtils.runIfNotDoneAndGet(futureTask);
+    }
+
+    @Test
+    void combineAll() {
+        FutureUtils.ConjunctFuture<Collection<String>> emptyConjunctFuture =
+                FutureUtils.combineAll(Collections.emptyList());
+        assertThatFuture(emptyConjunctFuture).isDone();
+        assertThatFuture(emptyConjunctFuture).eventuallySucceeds().asList().isEmpty();
+
+        CompletableFuture<String> future1 = new CompletableFuture<String>();
+        CompletableFuture<String> future2 = new CompletableFuture<String>();
+        FutureUtils.ConjunctFuture<Collection<String>> conjunctFuture =
+                FutureUtils.combineAll(Arrays.asList(future1, future2));
+
+        assertThatFuture(conjunctFuture).isNotDone();
+        assertThat(conjunctFuture.getNumFuturesTotal()).isEqualTo(2);
+        assertThat(conjunctFuture.getNumFuturesCompleted()).isZero();
+
+        future1.complete("Hello");
+        future2.complete("World");
+        assertThatFuture(conjunctFuture).eventuallySucceeds();
+
+        assertThatFuture(conjunctFuture).isDone();
+        assertThat(conjunctFuture.getNumFuturesTotal()).isEqualTo(2);
+        assertThat(conjunctFuture.getNumFuturesCompleted()).isEqualTo(2);
```

Comment:
assert the result string collection as well (this is the main usage of `combineAll()`)